nx-X11/config/cf/Server.tmpl

```diff
@@ -70,3 +70,11 @@ GLX_DEFINES = GlxDefines
 #ifdef XorgVersion
 #include <xorg.tmpl>
 #endif
+
+#ifdef HasXfont2
```

Comment:
Without our main Makefile, the nx-libs build defaults to libXfont1. This will die anyway, once we drop imake. It is a hack due to the limitations of imake, so we should not spend to much energy on this, IMHO.